index.html

```diff
@@ -13,16 +13,18 @@
 </head>
 <body>
     <header>
-        <div class="mylogo">My Logo</div>
-        <span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
-        </div>
+        
+            <div class="mylogo">My Logo</div>
+            <span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
+            </div>
+        
```

Comment:
- [ ] as mentioned by the previous reviewer kindly wrap your logo and hamburger button inside a nav tag, example : 
```HTML
<nav>
<div class="mylogo">My Logo</div>
<span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
</nav>
```